### PR TITLE
Add a timeout to Octopoes' XTDB client

### PR DESCRIPTION
**Pull request by originalsouth**
### Changes

Add a timeout to Octopoes' XTDB client

### Issue link

N/A

### Demo

N/A

### QA notes

Not needed

---

### Code Checklist

- [x] All the commits in this PR are properly PGP-signed and verified.
- [x] This PR only contains functionality relevant to the issue.
- [ ] I have written unit tests for the changes or fixes I made.
- [x] I have checked the documentation and made changes where necessary.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

- [x] Tickets have been created for newly discovered issues.
- [x] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [x] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

File: octopoes/octopoes/xtdb/client.py [hunk 1]
```diff
@@ -46,7 +46,10 @@ class XTDBStatus(BaseModel):
 @functools.cache
 def _get_xtdb_http_session(base_url: str) -> httpx.Client:
     return httpx.Client(
-        base_url=base_url, headers={"Accept": "application/json"}, transport=(httpx.HTTPTransport(retries=3))
+        base_url=base_url,
+        headers={"Accept": "application/json"},
+        transport=(httpx.HTTPTransport(retries=3)),
+        timeout=60,
```
Comment (zcrt) [inline octopoes/octopoes/xtdb/client.py:52]: I would like to suggest making the httpx timeouts an `env` variable

Comment (underdarknl) [reply octopoes/octopoes/xtdb/client.py:52]: I agree, look for similar options in other places.

Comment (dekkers) [reply octopoes/octopoes/xtdb/client.py:52]: I was already working on doing that in other places, so I can also include the xtdb client in my PR.

**Comment by originalsouth** [comment]
@dekkers will attend to this.

**Pull request closed without merging**